docs/developers/packaging.md

```diff
@@ -0,0 +1,298 @@
+(napari-packaging)=
+
+# Packaging
+
+Once a release is cut, napari is distributed in two main ways:
+
+* Packages: both to PyPI and conda-forge.
+* Installers: bundles that include napari plus its runtime dependencies in a step-by-step
+  executable.
+
+## Packages
+
+Despite its numerous dependencies, `napari` itself is a simple Python project that can be packaged
+in a straight-forward way.
+
+Creating and submitting the packages to PyPI (the repository you query when you do `pip install`)
+is handled in the [`.github/workflows/make_release.yml`][2] workflow. Creation is
+handled with `make dist` (as specified in our [`Makefile`][3]) and submission is done using the
+[official PyPA GitHub Action][4]. This workflow will also create a GitHub Release.
+
+Once the Python package makes it to PyPI, it will be picked by the `conda-forge` bots, which will
+automatically submit a PR to the [`napari-feedstock`][1] repository. This is all automated by the
+`conda-forge` infrastructure, so we only need to check that the metadata in the recipe has been
+adjusted for the new release. Pay special attention to the runtime dependencies and version strings!
```

Comment:
Could we add a link to the feedstock  https://github.com/conda-forge/napari-feedstock ?

And perhaps and example (merged PR) for reference like -> https://github.com/conda-forge/napari-feedstock/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed

Reply:
That's what the `[1]` there is doing :D I added a link to the examples though.